src/docs/asciidoc/getting_started/index.adoc

```diff
@@ -625,4 +625,93 @@ The result should be a 201 Http status.
 
 Look at the result in the UI, you should be able to see the card.
 
+=== Example 6: Sending cards from the UI
+
+==== Updating the perimeter
+In the top right corner, left to your username, there is the button to send a new card. 
+
+image::example6_button.png[example 6 screen shot]
+
+If you click on it now, you shouldn't be able to send a card because none of the groups you're part of has the right to send a card. So first we need to set up a new perimeter with the correct rights and associate it to one of your group.
+
+[source,JSON]
+----
+{
+	"id" : "example6-Perimeter",
+	"process" : "defaultProcess",
+	"stateRights" : [
+		{
+			"state" : "messageState",
+			"right" : "ReceiveAndWrite"
+		}
+	]
+}
+----
+Use the provided scripts :
+----
+./setupPerimeter.sh perimeter.json
+----
+
+==== Creating the template to send a card
+Now you need to update the bundle to include a template for the card sending screen. The bundle folder now contains usercard_message.handlebars:
+
+[source,handlebars]
+----
+<div class="opfab-textarea">
+	<label> MESSAGE </label>
+	<textarea id="message" name="message" placeholder="Write something.."
+		style="width:100%">{{card.data.message}}</textarea>
+</div>
+
+
+<script>
+    usercardTemplateGateway.getSpecificCardInformation = function () {
+        const message = document.getElementById('message').value;
+        const card = {
+        summary : {key : "message.summary"},
+		title : {key : "message.title"},
+        data : {message: message},
+        };
+        if (message.length<1) return { valid:false , errorMsg:'You must provide a message'}
+        return {
+            valid: true,
+            card: card
+        };
+
+    }
+
+</script>
+----
+
+You can package it and send it to the server with the provided scripts :
+
+----
+./packageBundle.sh | ./sendBundle.sh
+----
+
+Now if you reload the page, you should be able to send a card. Don't forget to send it to an entity you're a part of to receive it. For instance *Conctrol Center FR North*.
+
+image::example6_send_card.png[example 6 screen shot #2]
+
+==== Customize the card sending template
+To go further you can check out the folder *bundle_updated* to see a template where it is possible to add an object to the message similar to e-mails.
+
+You can see the updated UI using this command line and refreshing the page to update the bundle.
+
+----
+./packageBundle_updated.sh | ./sendBundle.sh
+----
+
+image::example6_send_card_object.png[example 6 screen shot #3]
+
+
+If you look at the *message.handlebars* file in the bundle, you will see how the template was changed to add the new object attribute.
+
+[source,handlebars]
+----
+<h2><b>Object:</b> {{card.data.object}}</h2>
+
+{{card.data.message}}
+----
+
```

Comment:
Add a link to https://opfab.github.io/documentation/current/reference_doc/#user_cards if user want more information on user cards